Makefile

```diff
@@ -127,7 +127,7 @@ deploy-app: ## Deploys the app to PaaS
 .PHONY: deploy-db-migration
 deploy-db-migration: ## Deploys the db migration app
```

Comment:
* Overrides are specified in manifest and vars files now
* `APPLICATION_NAME=db-migration` makes the generate manifest step creat with the new manifest.
* Leave no-route for safety